main/res/values/styles.xml

```diff
@@ -198,14 +198,26 @@
         <item name="android:autoText">true</item>
```

Comment:
"textinput_dropdown" seems to be the same as "textinput_edittext_singleline" currently. I support the idea to keep the separate style for the future, but couldn't it just be left empty with parent="textinput_edittext_singleline" for now?

Reply:
Inheritance is a bit different: `Widget.MaterialComponents.TextInputLayout.OutlinedBox.ExposedDropdownMenu` is parent for `textinput_dropdown`, whereas `Widget.MaterialComponents.TextInputLayout.OutlinedBox` (without `ExposedDropdownMenu`) is parent for `textinput_edittext`. 
I had tried using `textinput_dropdown` for the `EditText` definitions as well first, but that did not work for me, I did not get the colored border, so I started searching for other style templates and ended up with that one.

Reply:
You're right, I have overlooked that. Sorry, should have known...